fea-rs/src/compile/tables/var_store.rs

```diff
@@ -0,0 +1,312 @@
+//! Building the ItemVariationStore
+//!
+//! this will probably get moved to write-fonts at some point?
+
+use std::collections::HashMap;
+
+use indexmap::IndexMap;
+use write_fonts::tables::variations::{
+    ItemVariationData, ItemVariationStore, VariationRegion, VariationRegionList,
+};
+
+///
+/// this probably ends up living in write-fonts
+#[derive(Clone, Debug, Default)]
+pub(crate) struct VariationStoreBuilder {
+    // region -> index map
+    all_regions: HashMap<VariationRegion, usize>,
+    // we use an index map so that we have a determenistic ordering,
+    // which lets us write better tests
```

Comment:
If ordering is truly only for tests I would think the test should sort

Reply:
I would like to avoid having implementation details differ between testing and deployment where possible.